src/components/Landing/QAs.tsx

```diff
@@ -1,15 +1,16 @@
+/* eslint-disable react/jsx-key */
```

Comment:
disabled due to laziness

Reply:
would it be better to make this a warn rule, rather than disabling herfe so we are reminded to fix it? (I personally hate warnings so would motivate me to fix 😅 )

Reply:
fuck it, fixed it lol